packages/eslint-plugin-material-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-material-ui",
-  "version": "6.0.5",
+  "version": "6.3.0",
```

Comment:
The eslint dependency of this package has been updated. Proceeded with bumping the version just for the same of consistency. Do you agree?